src/SignalR/clients/csharp/Client/test/FunctionalTests/HubConnectionTests.cs

```diff
@@ -689,8 +689,7 @@ public async Task StreamAsyncCanBeCanceledThroughGetEnumerator(string protocolNa
                         }
                     });
 
-                    Assert.Single(results);
-                    Assert.Equal(0, results[0]);
+                    Assert.True(results.Count > 0 && results.Count < 1000);
```

Comment:
Do you know why this is needed here an not in StreamAsyncCanBeCanceled?

Reply:
It should be there as well. I'll add it